README.md

```diff
@@ -8,37 +8,173 @@ Ballerina AWS Redshift Connector
 [![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-aws.redshift/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-aws.redshift/actions/workflows/build-with-bal-test-native.yml)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
```

Comment:
Noticed that in the ReadMe.md, the `Build` and `GraalVM` status badges seem to appear broken. Is there a known reason for this issue?
<img width="835" alt="Screenshot 2024-01-09 at 06 54 18" src="https://github.com/ballerina-platform/module-ballerinax-aws.redshift/assets/39415471/ce31b955-1d8c-46c3-a71c-6920760e17f6">

Reply:
22a664e70074eb21cfc654bc4fc064038c087c5a